### PR TITLE
SUBMARINE-255. [PoC] Rename the submitter package name

### PR DESCRIPTION
**Pull request by jiwq**
### What is this PR for?
1. rename the package `org.apache.submarine.runtime.tony` to `org.apache.submarine.server.submitter.yarn`
2. rename the package `org.apache.submarine.runtime.yarnservice` to `org.apache.submarine.server.submitter.yarnservice`
3. update the conf in mini-submarine 
4. update the mini-submarine's doc

### What type of PR is it?
[Refactoring]

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-255

### How should this be tested?
https://travis-ci.org/jiwq/hadoop-submarine/builds/601290863

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: submarine-server/server-submitter/submitter-yarnservice/pom.xml [hunk 1]
```diff
@@ -1,39 +1,61 @@
...
   <properties>
     <!-- Needed for generating FindBugs warnings using parent pom -->
     <!--yarn.basedir>${project.parent.parent.basedir}</yarn.basedir-->
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>org.apache.submarine</groupId>
+      <artifactId>commons-runtime</artifactId>
+      <version>0.3.0-SNAPSHOT</version>
```
Comment (liuxunorg) [inline submarine-server/server-submitter/submitter-yarnservice/pom.xml]: Change `0.3.0-SNAPSHOT` to `${project.version}`, This is handled elsewhere.

File: submarine-server/server-submitter/submitter-yarnservice/pom.xml [hunk 2]
```diff
@@ -67,12 +89,21 @@
       <version>${commons.io.version}</version>
     </dependency>
 
-    <!-- Dependencies for Hadoop commons -->
```
Comment (liuxunorg) [inline submarine-server/server-submitter/submitter-yarnservice/pom.xml]: Why should I delete this comment?

File: submarine-server/server-submitter/submitter-yarn/pom.xml [hunk 3]
```diff
@@ -15,18 +15,17 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
-
-  <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>org.apache.submarine</groupId>
     <artifactId>server-submitter</artifactId>
     <version>0.3.0-SNAPSHOT</version>
-    <relativePath>../pom.xml</relativePath>
```
Comment (liuxunorg) [inline submarine-server/server-submitter/submitter-yarn/pom.xml]: Why should I delete this line?

File: submarine-server/server-submitter/submitter-yarn/src/main/java/org/apache/submarine/server/submitter/yarn/Utils.java [hunk 4]
```diff
@@ -36,10 +35,10 @@
 import static org.apache.submarine.commons.runtime.conf.SubmarineConfiguration.SUBMARINE_RUNTIME_APP_TYPE;
 
 /**
- * Utilities for Tony Runtime.
+ * Utilities for YARN Runtime.
  */
-public final class TonyUtils {
-  private static final Log LOG = LogFactory.getLog(TonyUtils.class);
+public final class Utils {
```
Comment (liuxunorg) [inline submarine-server/server-submitter/submitter-yarn/src/main/java/org/apache/submarine/server/submitter/yarn/Utils.java]: Modify `Utils` to `YarnUtils`?

File: submarine-server/server-submitter/submitter-yarnservice/src/test/java/org/apache/submarine/server/submitter/yarnservice/utils/TestClassPathUtilities.java [hunk 5]
```diff
@@ -6,29 +6,29 @@
...
 import org.junit.After;
+import org.junit.Assert;
 import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.Test;
 
 import java.io.File;
 import java.io.IOException;
 
-import static org.junit.Assert.assertEquals;
```
Comment (liuxunorg) [inline submarine-server/server-submitter/submitter-yarnservice/src/test/java/org/apache/submarine/server/submitter/yarnservice/utils/TestClassPathUtilities.java]: Don't make this kind of meaningless modification.

Comment (jiwq) [reply submarine-server/server-submitter/submitter-yarnservice/src/test/java/org/apache/submarine/server/submitter/yarnservice/utils/TestClassPathUtilities.java]: OK, I had resolved it.

File: submarine-server/server-submitter/submitter-yarn/pom.xml (hunk 3 above)
Comment (jiwq) [reply submarine-server/server-submitter/submitter-yarn/pom.xml]: Sorry for that, I had resolved it.

File: submarine-server/server-submitter/submitter-yarn/src/main/java/org/apache/submarine/server/submitter/yarn/Utils.java (hunk 4 above)
Comment (jiwq) [reply submarine-server/server-submitter/submitter-yarn/src/main/java/org/apache/submarine/server/submitter/yarn/Utils.java]: Good, I had resoled it.

File: submarine-commons/commons-utils/src/main/java/org/apache/submarine/commons/utils/NetUtils.java [hunk 6]
```diff
@@ -1,18 +1,20 @@
 /*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
+ * Licensed to the Apache Software Foundation (ASF) under one
```
Comment (liuxunorg) [inline submarine-commons/commons-utils/src/main/java/org/apache/submarine/commons/utils/NetUtils.java]: Please do not modify the content that is not related to this PR.

Comment (jiwq) [reply submarine-commons/commons-utils/src/main/java/org/apache/submarine/commons/utils/NetUtils.java]: Done

**Comment by yuanzac** [comment]
Thanks @jiwq and @liuxunorg 
LGTM

**Pull request closed without merging**